index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Mobile Version Skeleton</title>
+</head>
+
+<body>
+    <section class="container">
+        <section class="pg-1">
+            <header>
+                <img class="img" src="img/time.png" alt="time">
+
+                <nav>
+                    <ul>
+                        <li><img src="img/cellular.png" alt="cellular"></li>
+                        <li><img src="img/wifi.png" alt="wifi"></li>
+                        <li><img src="img/capacity.png" alt="capacity"></li>
+                    </ul>
+                </nav>
+            </header>
+
+            <div class="logo">
+
+                <h2 class="my-logo">My Logo</h2>
+
+                <img class="ham" src="img/hamburger.png" alt="hamburger">
+
+            </div>
+
+            <div class="description">
+
+                <div class="name">
+                    <h2>I’m Mariane <br> Glad to see <br> you!</h2>
```

Comment:
Please ensure that the main title (e.g., "I'm Marianne ...") is a `h1` tag.

Reply:
@micheaol  Thanks for your suggestion. Please the requirements suggest h2 ta
![Screenshot 2022-02-10 at 09 33 55](https://user-images.githubusercontent.com/22225878/153378897-e0c36561-6c56-4e16-8cdc-a374da1f15dc.png)
g

Reply:
@micheaol am sorry it doesn't give me the right styling when I used H1 and if you can share a copy of the requirements with me, I would be glad to go through thanks

Reply:
Please find below as requested.
![image](https://user-images.githubusercontent.com/10332499/153419125-c1981fab-20b1-479b-a984-6309ef0e442b.png)

Reply:
@micheaol 
Thanks for the image. I appreciate and am sorry for the stress we didn't have the requirements to work with and was asked to work with the Figma diagram.

it has been implemented accordingly please review.